src/fides/api/task/manual/manual_task_utils.py

```diff
@@ -283,8 +284,40 @@ def get_manual_task_instances_for_privacy_request(
     )
 
 
+def filter_manual_task_configs_by_policy(
+    policy: "Policy", config: ManualTaskConfig
+) -> bool:
+    """
+    Filter manual task configs by policy.
+    """
+    filter_match = False
+    has_access_rules = bool(policy.get_rules_for_action(action_type=ActionType.access))
+    has_erasure_rules = bool(
+        policy.get_rules_for_action(action_type=ActionType.erasure)
+    )
+
+    if has_access_rules and has_erasure_rules:
+        # If both access and erasure rules exist, include both types
+        if config.config_type in [
+            ManualTaskConfigurationType.access_privacy_request,
+            ManualTaskConfigurationType.erasure_privacy_request,
+        ]:
+            filter_match = True
+    elif has_access_rules:
+        # Only access rules - only include access configurations
+        if config.config_type == ManualTaskConfigurationType.access_privacy_request:
+            filter_match = True
+    elif has_erasure_rules:
+        # Only erasure rules - only include erasure configurations
+        if config.config_type == ManualTaskConfigurationType.erasure_privacy_request:
+            filter_match = True
+
+    return filter_match
```

Comment:
I'm not saying you should do this, because this is already tested and working, and its fine as is, but if you wanted to make this function a bit smaller, I think this would do the same? (haven't tested, these if policy statements might need to be wrapped in bool() or kept as defined above if you prefer that style)

    allowed: set[ManualTaskConfigurationType] = set()
    if policy.get_rules_for_action(action_type=ActionType.access:
        allowed.add(ManualTaskConfigurationType.access_privacy_request)
    if policy.get_rules_for_action(action_type=ActionType.erasure):
        allowed.add(ManualTaskConfigurationType.erasure_privacy_request)
    return config.config_type in allowed